app/src/main/res/values-it/strings.xml

```diff
@@ -272,35 +272,35 @@
     <string name="pesdk_editor_text_exportProgressUnknown">Esportazione…</string>
     <string name="pesdk_editor_text_exportProgress" formatted="false">Esportazione %s.</string>
     <string name="pesdk_sticker_title_name">Sticker</string>
-    <string name="pesdk_sticker_title_color">Sticker Color</string>
-    <string name="pesdk_sticker_title_options">Sticker Options</string>
-    <string name="pesdk_sticker_button_add">Add</string>
-    <string name="pesdk_sticker_button_color">Color</string>
-    <string name="pesdk_sticker_button_delete">Delete</string>
-    <string name="pesdk_sticker_button_bringToFront">To Front</string>
-    <string name="pesdk_sticker_button_straighten">Straighten</string>
-    <string name="pesdk_sticker_button_replace">Replace</string>
-    <string name="pesdk_sticker_button_opacity">Opacity</string>
-    <string name="pesdk_sticker_button_contrast">Contrast</string>
-    <string name="pesdk_sticker_button_saturation">Saturation</string>
-    <string name="pesdk_sticker_button_brightness">Brightness</string>
+    <string name="pesdk_sticker_title_color">Colore Sticker</string>
+    <string name="pesdk_sticker_title_options">Opzioni Sticker Options</string>
+    <string name="pesdk_sticker_button_add">Aggiungi</string>
+    <string name="pesdk_sticker_button_color">Colore</string>
+    <string name="pesdk_sticker_button_delete">Cancella</string>
+    <string name="pesdk_sticker_button_bringToFront">Sposta davanti</string>
+    <string name="pesdk_sticker_button_straighten">Raddrizza</string>
+    <string name="pesdk_sticker_button_replace">Sostituisci</string>
+    <string name="pesdk_sticker_button_opacity">Opacità</string>
+    <string name="pesdk_sticker_button_contrast">Contrasto</string>
+    <string name="pesdk_sticker_button_saturation">Saturazione</string>
+    <string name="pesdk_sticker_button_brightness">Luminosità</string>
     <string name="pesdk_sticker_category_name_custom">Uploads</string>
```

Comment:
@tibbi Would it be possible to have a bit of context for this line? I'm not sure what it means here

Reply:
sticker category which doesnt contain preset stickers, but custom ones added by the user